compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeDialog.desktop.kt

```diff
@@ -64,6 +65,14 @@ class ComposeDialog(
         content = content
     )
 
+    // TODO(CL) non-experimental new API. we can't remove it when we merge it into AOSP, we can just deprecate it.
```

Comment:
Cannot it be internal?

Reply:
No, otherwise third-party developers can't pass composition locals. I already know, that Aurora uses it.

Reply:
Maybe mark it as experimental at least?

Reply:
We can deprecate it, if realize it is not we want in the future.

Better to provide backward-compatible API for the libraries